SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Storage/StorageAdapter.cs

```diff
@@ -117,21 +226,10 @@ public Task InitializeStorageAsync()
                         throw new StorageException("Cannot initialize SQLite library.", e);
                     }
                 }
-
-                // In SQLite-net 1.5.231 constructor parameters were changed.
-                // Using reflection to accept newer library version.
-                _dbConnection = (SQLiteAsyncConnection)typeof(SQLiteAsyncConnection)
-                    .GetConstructor(new[] { typeof(string), typeof(bool) })
-                    ?.Invoke(new object[] { _databasePath, true });
-                if (_dbConnection == null)
-                {
-                    _dbConnection = (SQLiteAsyncConnection)typeof(SQLiteAsyncConnection)
-                        .GetConstructor(new[] { typeof(string), typeof(bool), typeof(object) })
-                        ?.Invoke(new object[] { _databasePath, true, null });
-                }
-                if (_dbConnection == null)
+                raw.SetProvider(new SQLite3Provider_e_sqlite3());
```

Comment:
Wrap it to try/catch. We had reports that it can fail if project uses SQLite itself